### ISSUE TITLE
Libraries/Frameworks not signed properly

### ISSUE THREAD

**Issue by newkind** [rileytestut/AltStore#159]
There is a possibility that libraries/frameworks aren't signed properly on sideloaded apps and this can lead to app crashes ie. in Kodi.

Here's a topic at Kodi forum : https://forum.kodi.tv/showthread.php?tid=349007&pid=2940993#pid2940993

Related tweet: https://twitter.com/App_Valley_vip/status/1246350190262484992

**Comment by UprisingPulse** [comment]
Yes I also tried with many different versions of kodi but nothing works

Hope Riley take a look

**Comment by gqbre** [comment]
Still unfixed

**Comment by gqbre** [comment]
Keep Watching @rileytestut

**Comment by TheQL** [comment]
AltStore just crashes when trying to sideload Kodi.ipa - not sure if related. Same happened with nullximpactorlite.

**Comment by th3cube** [comment]
Any hope this will be fixed soon?

**Comment by Pakistaniboy442** [comment]
Of course, same thing in 2019.

**Comment by jorin21** [comment]
Hey guys, just want to know if this is still being looked at thank you!

**Comment by jax-tr** [comment]
Hi! Is this still being worked on? Thank you!
